src/AppBundle/Membership/AdherentChangePasswordHandler.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace AppBundle\Membership;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface;
+
+class AdherentChangePasswordHandler
+{
+    private $manager;
+    private $encoderFactory;
+
+    public function __construct(ObjectManager $manager, EncoderFactoryInterface $encoderFactory)
```

Comment:
Because the factory already holds every instances, no need to create an extra service as done in other places.